herddb-core/src/main/java/herddb/core/TableManager.java

```diff
@@ -401,6 +413,18 @@ public Table getTable() {
                     return table;
                 }
             };
+        } else if (table.getColumn(table.primaryKey[0]).type == ColumnTypes.NOTNULL_LONG) {
```

Comment:
can you merge this branch with the ColumnTypes.LONG one ?

Reply:
👍